app/src/main/java/com/github/epfl/meili/forum/NewPostViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.epfl.meili.forum
```

Comment:
What do you think about having the PostViewModel, ForumViewModel & NewPostViewModel merged into one ViewModel since they are using the same service? But again, they are used for different activities. I am just curious to know your opinion about this.

Reply:
My guess is that they will vary a lot once we add more functionality to the forum, such as comments, upvotes, sorting methods and all that, so I think its better to keep them separate.